### ISSUE TITLE
Audacity logo used in Windows installation dialogs

### ISSUE THREAD

**Issue by petersampsonaudacity** [tenacityteam/tenacity-legacy#502]
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

latest master build for Tenacity  https://github.com/tenacityteam/tenacity/actions/runs/1145691475

### Describe the bug.

1. Install latest master build
2. Observe Audacity Logo and not Tenacity in the installation dialog
![image](https://user-images.githubusercontent.com/50205984/130107613-63bd6b0f-99b5-4a1d-ac30-76820efc0674.png)

3. Next dialog window
![image](https://user-images.githubusercontent.com/50205984/130108044-5eb21ee6-6360-4ea0-9e87-aaae13696979.png)
Not only Audacity logo - bit many mentions of "Audacity"

4. Next dialog window
![image](https://user-images.githubusercontent.com/50205984/130108254-89fbad17-67e4-4c47-8c96-1e02a0429a06.png)

5. And the next dialog window
![image](https://user-images.githubusercontent.com/50205984/130108368-867520ac-d71b-46ca-8cc1-9db285fccd71.png)

6. and the next
![image](https://user-images.githubusercontent.com/50205984/130108437-7d41e512-1e93-41de-a6cd-6a14c79ff416.png)

7. and this one has audacity references (more than can be seen here)
![image](https://user-images.githubusercontent.com/50205984/130108822-870240a4-6e9a-420b-8516-0df3aa2a5403.png)

8. and now the final installation window with a big Audacity icon
![image](https://user-images.githubusercontent.com/50205984/130108970-ef8fd086-4077-495d-8d9c-f1ea078fd317.png)

### Expected behavior

Tenacity logos and Tenacity references throughout

### OS

Windows 10

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by Be-ing** [comment]
Thanks. We're aware we still have work to do removing Audacity trademarks before we can do a release.

**Comment by petersampsonaudacity** [comment]
Thanks for the "Good first issue" - but it's actually not my first, I also posted this one about the Audacity Manual:
https://github.com/tenacityteam/tenacity/issues/353
_This one should also be flagged as a Release Blocker not least because of the many "?" help buttons in the app that link to the manual._

And this one re Tenacity and Audacity blocking each other:
https://github.com/tenacityteam/tenacity/issues/364

**Comment by Be-ing** [comment]
The "good first issue" label means that this is a good issue for a new contributor to fix as their first contribution; it's not about who reported the issue.

**Comment by Paul-Licameli** [comment]
We love you @petersampsonaudacity 

A world of nothing but coders like me would be a very dull world indeed.

**Comment by NoobMaster1999** [comment]
Hi. is the issue still open? I want to solve this issue..
Because it is my first issue.

**Comment by Be-ing** [comment]
Yes, that would be great if you replaced the Audacity logo in the Windows installer with the Tenacity logo. SVGs for the logo are in https://github.com/tenacityteam/assets

**Comment by NoobMaster1999** [comment]
@Be-ing please assign me for this

**Comment by petersampsonaudacity** [comment]
Tested on macOS 11.5.2 Big Sur with latest master build.

There is a similar problem with macOS DMG installet
<img width="1226" alt="Screenshot 2021-08-21 at 13 48 54" src="https://user-images.githubusercontent.com/50205984/130322393-311e034b-8c35-4e31-93b0-4a9869b1396f.png">

**Comment by Be-ing** [comment]
That is tracked separately in #450.

**Comment by st0chast** [comment]
I tried searching for `Forum` or `Wiki` on https://hosted.weblate.org/projects/tenacity/tenacity/en/ and those lines from the installer seem to be absent. I know it's not necessary to fix this issue quickly, but I felt like this had to be mentioned.

**Comment by petersampsonaudacity** [comment]
Testing on W10 with @FugiMuffi 's build https://github.com/tenacityteam/tenacity/actions/runs/1154630604

This properly replaces the Audacity logo throughout the installation process with the Tenacity logo.

But there is still a dialog window in the installation process that has references and links to Audacity and it's websites - I will log this as a separate issue (as the title of this issue will be resolved by pulling @FugiMuffi 's fix).
